backend/src/data/repositories/mentees-to-mentors/mentees-to-mentors.repository.ts

```diff
@@ -110,6 +110,21 @@ class MenteesToMentors {
       )
       .execute();
   }
+
+  public async checkIsMentorForMentee({
+    courseId,
+    menteeId,
+    mentorId,
+  }: MenteesToMentorsRequestDto): Promise<boolean> {
+    const menteeToMentor = await this.#MenteesToMentorsModel
+      .query()
+      .where({ courseId })
+      .andWhere({ menteeId })
+      .andWhere({ mentorId })
+      .first();
```

Comment:
```
 .query()
 .select(1)
 .where({ courseId })
 .andWhere({ menteeId })
 .andWhere({ mentorId })
.first();
```

Reply:
Don't quite understand the purpose of it, does it give us a better performance?

Reply:
@fshabanov without select(1) the query will return all the fields - which are not necessary here, select(1) is useful in cases when you just need to check whether there is an instance in db which satisfies your conditions in where clause, and it didn't extract the fields from the table

Reply:
@kalinkaaaa14 Yeah, just wanted to clarify, if we don't select the columns and just return 1, does it improve a performance or is it just a common way to do it?

Reply:
It's a common optimization practice to check if record exists, it's obvious that in our case orm would need less effort to map query results, but in general it's not critical for a such simple request (as we turn a blind eye to many queries where we can avoid nested subqueries usage)

Reply:
Got it, thanks!